src/js/components/NewTodo.js

```diff
@@ -35,20 +34,30 @@ export default class NewTodo extends Component {
     render() {
         if (!this.state.isNewTodoFormDisplayed) {
             return (
```

Comment:
Avoid using in render two `return`. Maybe better would be creating one `return` and to make comparision You could use that operator ` ? : `